specification/streamanalytics/resource-manager/Microsoft.StreamAnalytics/preview/2017-04-01-preview/inputs.json

```diff
@@ -594,6 +594,22 @@
         }
       }
     },
+    "RawStreamInputDataSource": {
+      "description": "Describes a raw input data source. This data source type is only applicable/usable when using the query testing API. You cannot create a job with this data source type or add an input/output of this data source type to an existing job.",
+      "x-ms-discriminator-value": "Raw",
+      "allOf": [
+        {
+          "$ref": "#/definitions/StreamInputDataSource"
+        }
+      ],
+      "properties": {
+        "properties": {
+          "$ref": "#/definitions/RawInputDatasourceProperties",
+          "description": "The properties that are associated with a raw input.",
```

Comment:
NIT: Usually in descriptions, if the property is required during create, we put it in the description. You may see the following string "Required on PUT (CreateOrReplace) requests." scattered through the swagger specs